### DISCUSSION TITLE
API Reference documentation

### DISCUSSION THREAD

**Discussion by soslan** [fastapi/fastapi#6698] [Questions] [answered]
Does FastAPI has an API Reference? Like [this](https://flask.palletsprojects.com/en/2.0.x/api/).

**Comment by Kludex** [comment]
No.
Duplicated #804
Early implementation and @tiangolo 's thoughts: #2275

**Comment by YuriiMotov** [comment, marked as answer]
https://fastapi.tiangolo.com/reference/
